bytecode/concrete.py

```diff
@@ -280,11 +280,7 @@ def _assemble_lnotab(first_lineno, linenos):
         return b"".join(lnotab)
 
     @staticmethod
-    def _pack_linetable(doff, dlineno):
-        linetable = []
-        while doff > 254:
-            linetable.append(b"\xfe\x00")
-            doff -= 254
+    def _pack_linetable(doff, dlineno, linetable):
```

Comment:
I would prefer for linetable to be the first argument TBH.
```suggestion
    def _pack_linetable(linetable, doff, dlineno):
```

Reply:
Ok